cloudferry/cli/v1/commands.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) 2016 Mirantis Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied.
+# See the License for the specific language governing permissions and#
+# limitations under the License.
+
+from fabric.api import env
+
+from cloudferry.cli import base
+from cloudferry.cloud import cloud_ferry
+from cloudferry.cloud import grouping
+from cloudferry.lib.scheduler import scenario
+
+env.forward_agent = True
+env.user = 'root'
```

Comment:
Looks like we do not need it at all, because we always define the user for `api.run()` command.